cmd/gossamer/main.go

```diff
@@ -234,6 +234,14 @@ func gossamerAction(ctx *cli.Context) error {
 		return err
 	}
 
+	if cfg.Global.RetainBlocks < 256 {
```

Comment:
maybe make 256 a constant `DefaultRetainBlocks` since it's used a lot

Reply:
Done